packages/babel-helpers/src/helpers.js

```diff
@@ -4,9 +4,9 @@ let helpers = {};
 export default helpers;
 
 helpers.typeof = template(`
-  (function (obj) {
-    return obj && typeof Symbol !== "undefined" && obj.constructor === Symbol ? "symbol" : typeof obj;
-  });
+  (typeof Symbol === "function" && typeof Symbol() === "symbol")
```

Comment:
Forgive me for lacking context here.

Does this mean every use of typeof will call Symbol() or does this result get cached and the resulting function is used?